runat.js

```diff
@@ -22,20 +26,17 @@ function WorkQueue(options) {
 }
 inherits(WorkQueue, Duplex)
 
-WorkQueue.prototype._connect = function _connect() {
-  if (this.client == null)
-    this.client = redis.createClient(this.options.redisPort, this.options.redisHost)
-}
-
 WorkQueue.prototype._write = function _write(job, encoding, callback) {
-  if (!job.key)
+  var self = this
```

Comment:
good catch! Maybe throw `'use strict'` at the top of the file, apparently this pre-dates my using strict mode all the time! :blush: